src/control-plane/backend/lambda/api/model/stacks.ts

```diff
@@ -670,7 +671,7 @@ export class CDataProcessingStack extends JSONObject {
   })
     ScheduleExpression?: string;
 
-  @JSONObject.optional('')
+  @JSONObject.optional(TRANSFORMER_AND_ENRICH_CLASS_NAMES)
```

Comment:
@tyyzqmf @jingnanl  Will this change impact updating the existing v1.0.x pipeline?

Reply:
I don't think this change may impact updating existing pipeline. I've deployed this change to my local env and tested with existing pipeline. The existing enrichments were not changed.

If I edited the pipeline to deselect any enrichment, the pipeline will remain with no enrichment. @amliuyong is it expected behavior?

Reply:
the existing pipeline should use the new value as the class names.  Just note that, the transformer default value is configured in below file:
src/control-plane/backend/config/dictionary.json

changed it accordingly now.
 
https://github.com/awslabs/clickstream-analytics-on-aws/pull/297/commits/689462318a2ea92c5c1d95960ec8c0d9e3e911d5#diff-7157d9f0404a55e405c2ca2c07feec2a6f5c2b73e7b58b40428376a94ca674c0R35-R37

Reply:
> If I edited the pipeline to deselect any enrichment, the pipeline will remain with no enrichment. @amliuyong is it expected behavior?

Yes, the customer can disable the built-in enrich plug-ins.